tox.ini

```diff
@@ -70,18 +70,11 @@ line_length=88
 deps = pylint
 commands = pylint --rcfile=tox.ini src/diepvries
 
-# pylint
-[MESSAGES CONTROL]
-disable =
-    duplicate-code
-
-[BASIC]
+[pylint]
+disable=duplicate-code
```

Comment:
Ah yes, some of those changes are also reflected in #32. We should align on merging both PRs as they update pylint's configuration

Reply:
I like the idea of changing `[MESSAGES CONTROL]` to `## MESSAGES CONTROL` so I'll align the `[pylint]` section to what's written there.